pypeit/scripts/show_2dspec.py

```diff
@@ -40,6 +38,8 @@ def parser(options=None):
     parser.add_argument('--det', default=1, type=int, help='Detector number')
     parser.add_argument('--showmask', default=False, help='Overplot masked pixels',
                         action='store_true')
+    parser.add_argument('--clean', default=False, help="Remove trace plotting",
+                        action = "store_true")
```

Comment:
Just an opinion, but `clean` feels non-intuitive.  As a user, I'm not sure I would know what you mean by "Remove" trace plotting.  It's just that you're not overplotting the trace data, right?

Reply:
changed clean to removetrace to make it clearer